src/reference/crash-reporting.md

```diff
@@ -35,7 +38,7 @@ $ flutter config --no-analytics
 ```
```

Comment:
Additionally, we should change the command they need to run to
```
flutter --disable-telemetry
``` 
The command above will work for both the old and new unified analytics package but the PDD states we should use the one above

And to re-enable incase they want to is below
```
flutter --enable-telemetry
```

Reply:
Thanks, updated! Will `flutter config` still work?

Reply:
Perfect, and yep that will still work so we don't break any previous users but we should encourage the use of the new command for telemetry related stuff